internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -505,38 +543,65 @@ func (mc *mac) triggerUserNew(w http.ResponseWriter, r *http.Request, lines []st
 		w.Write([]byte("false"))
 		return false
 	}
-
+	
 	for _, userWebhook := range userWebhooks {
-		if userWebhook.Url == "" {
-			glog.Errorf("User webhook %s has no URL", userWebhook.ID)
-			continue
-		}
-		glog.V(model.DEBUG).Infof("Calling playback.user.new webhook %s", userWebhook.Url)
+		
+			if userWebhook.Url == "" {
+				glog.Errorf("User webhook %s has no URL", userWebhook.ID)
+				return true
+			}
```

Comment:
I guess we can skip this check now that we are parsing the URL below?

Reply:
Nevermind, go does not return an error when you try to parse an empty string as a URL. I think this check might still be unnecessary tho, since the API is the one doing this kind of validation of the resources. We can assume they are valid if they exist, I guess.